doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -132,31 +132,15 @@ As a tenant developer, you can create new client-side EP schemas for your applic
 
```

Comment:
In "creating a new CT" section you should refer to the corresponding section in CTL page.